### DISCUSSION TITLE
Version questions

### DISCUSSION THREAD

**Discussion by janhaertel** [jquery/jquery-ui#2285] [Q&A] [answered]
Will there be a **1-14-stable** branch?
Will the version in the **package.json** be updated to **1.14**?
At the moment it is set to **1.14.1-pre**.

**Comment by mgol** [comment, marked as answer]
This is not a bug report, moving to discussuons.

`1.14` is not a valid semver version. `1.14.1-pre` is used because what's on `main` is unstable; in production you should use real versions.
